### ISSUE TITLE
Replace the middleware URL from `connect.woocommerce.com` to `api.woocommerce.com/integrations`

### ISSUE THREAD

**Issue by ibndawood** [woocommerce/woocommerce-gateway-paypal-express-checkout#896]
The `connect.woocommerce.com` server is due to be retired. The `api.woocommerce.com`  server now provides the connect service. The base URL will now be `api.woocommerce.com/integrations`. All other endpoints should now be accessible from the base URL.

## Endpoints

- [x] `/login/ppe`
- [x] `/auth/ppe`
- [x] `/renew/ppe`
- [x] `/revoke/ppe`
- [x] `/login/ppesandbox`
- [x] `/auth/ppesandbox`
- [x] `/renew/ppesandbox`
- [x] `/revoke/ppesandbox`

**Comment by wjrosa** [comment]
I have created a new instance of WooCommerce and installed the plugin. Then, I manually changed the base URL for the new one above (`/woocommerce-gateway-paypal-express-checkout/includes/class-wc-gateway-ppec-ips-handler.php#12`) and tested the authentication endpoints (/login and /auth) using my PayPal account with the help of @ibndawood. Everything worked as expected.
As for the other endpoints, I could not find any reference to them on the extension code. But I tested those manually using Postman and got the expected result (400 status, not implemented).
